ReactiveSwift/Scheduler.swift

```diff
@@ -144,7 +144,7 @@ public final class QueueScheduler: DateSchedulerProtocol {
 		return Date()
 	}
 
-	internal let queue: DispatchQueue
+	public let queue: DispatchQueue
```

Comment:
Is this supposed to be public now? It's leaking a huge implementation detail, and users could dispatch things manually here.